frontend/components/views/legal/TermsOfService.vue

```diff
@@ -174,9 +174,41 @@
           law enforcement if it deems necessary.
         </p>
 
-        <h2>Bank Transfers</h2>
-        <p>You expressly authorize Artconomy's service provider, Dwolla, Inc. to originate credit transfers
-          to your financial institution account.</p>
+        <h3>Scope, Intent, and Responsibilities</h3>
+
+        <p>
+          Artconomy's purpose is to provide a marketplace and set of tools to improve the business viability of artists
+          creating custom commission work. As a marketplace, it intends also to connect commissioners with artists.
+          In order to provide this service, users are expected to observe the following:
+        </p>
+
+        <ul>
+          <li>
+            Users offering products for sale of art created by another entity must include that entity's trade name
+            in the relevant product descriptions. This includes human artists, artist collectives, and AI programs which
+            contribute substantially to the final products being delivered.
+          </li>
+          <li>
+            Users are expected to make a best effort to rate and tag content according to the content ratings as
+            described on the relevant FAQ pages. These pages are available at
+            <a href="https://artconomy.com/faq/other/content-ratings">https://artconomy.com/faq/other/content-ratings</a>
+            and
+            <a href="https://artconomy.com/faq/other/tagging">https://artconomy.com/faq/other/tagging</a>.
```

Comment:
This link won't work until this PR is deployed, but the contents are found earlier in this PR.